server/insertPost.js

```diff
@@ -1,35 +1,19 @@
 #!/usr/bin/env node
 "use strict"
 
-const fs       = require("fs")
-const path     = require("path")
-const config   = require("../config")
+const fs        = require("fs")
+const denodeify = require("denodeify")
+const path      = require("path")
+const config    = require("../config")
 // const shortid  = require("shortid")
 // const utils    = require("./utils")
 const logger   = require("./logger")
 
-let prompt
+let prompt = undefined
 const globals = {}
 globals.coll = undefined
 globals.path = undefined
 
-function denodeify(fn, args, alt, th) {
```

Comment:
at the very least I can pull them into `utils` and probably rename it to avoid confusion